slave-ansible/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM openshift/jenkins-slave-base-centos7
+FROM docker.io/fhwendy/jenkins-slave-base-centos7:RHMAP15237
 
 MAINTAINER Oleg Matskiv <omatskiv@redhat.com>
```

Comment:
@AdamSaleh What I mean is this `MAINTAINER` line in particular. I think that might result in an additional image layer, just like a `RUN` or `CMD` or whatever other dockerfile commands are available. It's probably fine anyway, you can ignore me. :)